RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -307,6 +310,17 @@ private static PluginManifest CreatePluginDefinition(IPackage package)
             pluginDefn.Organization = GetValidManifestString(ListToString(package.Owners));
             pluginDefn.Version = GetValidManifestString(package.Version.ToNormalizedString());
 
+            if (package.LicenseUrl != null)
+            {
+                // The TermsConditionsUrl is only displayed in the "Update Center - Available" page
+                // i.e. for plugins that are available through the public Update Center.
+                // If the property has a value then the link will be displayed with a checkbox 
+                // for acceptance.
+                // It is not used when plugins are directly dropped into the extensions\plugins
+                // folder of the SonarQube server.
+                pluginDefn.TermsConditionsUrl = package.LicenseUrl.ToString();
```

Comment:
There's a straight mapping between the plugin definition and the NuGet package LicenseUrl property so this should have been set as part of [SFSRAP-20](https://jira.sonarsource.com/browse/SFSRAP-20). However, I'll fix it under [SFSRAP-22](https://jira.sonarsource.com/browse/SFSRAP-22) as that issue specifically refers to this property.
Separately, I'll create one or more new issues to deal with other aspects of the licensing experience for v1.